### ISSUE TITLE
Watch was unable to register: javax.net.ssl.SSLHandshakeException: connection closed

### ISSUE THREAD

**Issue by brianjmurrell** [home-assistant/android#4438]
**Home Assistant Android app version(s):**
Phone: 2024.4.1-full
Watch: 2024.4.1

**Android version(s):**
Phone: 14
Watch: ??  (whatever is on a Pixel Watch 2)

**Device model(s):**
Phone: OnePlus 6
Watch: Pixel Watch 2

**Home Assistant version:**
2024.5.3

**Last working Home Assistant release (if known):**
N/A

**Description of problem, include YAML if issue is related to notifications:**
Trying to connect my watch to HA I select my server on the watch and told to continue on the phone which I do, entering my account and password and then the app says _Watch was unable to register_.

**Companion App Logs:**
[ BTW: it's pretty nasty to ask for these logs without warning the user that they are revealing their location by doing it ]
```
--------- beginning of main
06-04 16:20:06.226  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: true, expireMillis: 1717532394702, currentMillis: 1717532406226)
06-04 16:20:06.276  6912  6912 D VRI[SettingsActivity]: Start draw after previous draw not visible
06-04 16:20:06.496  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: true, expireMillis: 1717532394702, currentMillis: 1717532406496)
06-04 16:20:06.496  6912  6912 D IntegrationRepository: setAppActive(): true
06-04 16:20:06.768  6912  6958 I mpanion.android: NativeAlloc concurrent copying GC freed 61MB AllocSpace bytes, 127(4352KB) LOS objects, 75% free, 18MB/72MB, paused 105us,65us total 253.463ms
06-04 16:20:07.100  6912  4334 D WearOnboardingListener: onMessageReceived: MessageEventParcelable[17557,/request_home_assistant_instance, size=0]
06-04 16:20:07.101  6912  4334 D WearOnboardingListener: sendHomeAssistantInstance: 46ffdd4b
06-04 16:20:07.105  6912  4334 D ServerConnectionInfo: Using external URL
06-04 16:20:07.111  6912  6912 D WearOnboardingListener: sendHomeAssistantInstance: success
06-04 16:20:08.004  6912  6912 W Choreographer: Frame time is 0.008233 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
06-04 16:20:14.042  6912  6912 D LocBroadcastReceiver: Received location update.
06-04 16:20:14.054  6912  6912 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
06-04 16:20:14.056  6912  7032 D LocBroadcastReceiver: Last Location:
06-04 16:20:14.056  6912  7032 D LocBroadcastReceiver: Coords:([redacted])
06-04 16:20:14.056  6912  7032 D LocBroadcastReceiver: Accuracy: 100.0
06-04 16:20:14.056  6912  7032 D LocBroadcastReceiver: Bearing: 0.0
06-04 16:20:14.057  6912  7032 D LocBroadcastReceiver: Begin evaluating if location update should be skipped
06-04 16:20:14.057  6912  7032 D LocBroadcastReceiver: Received location that is 90 milliseconds old, 1717532413967 compared to 1717532414057 with source fused
06-04 16:20:14.057  6912  7032 D LocBroadcastReceiver: Duplicate location received, not sending to HA
06-04 16:21:14.025  6912  6912 D LocBroadcastReceiver: Received location update.
06-04 16:21:14.041  6912  6912 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
06-04 16:21:14.048  6912  7032 D LocBroadcastReceiver: Last Location:
06-04 16:21:14.048  6912  7032 D LocBroadcastReceiver: Coords:([redacted])
06-04 16:21:14.048  6912  7032 D LocBroadcastReceiver: Accuracy: 100.0
06-04 16:21:14.048  6912  7032 D LocBroadcastReceiver: Bearing: 0.0
06-04 16:21:14.051  6912  7032 D LocBroadcastReceiver: Begin evaluating if location update should be skipped
06-04 16:21:14.051  6912  7032 D LocBroadcastReceiver: Received location that is 84 milliseconds old, 1717532473967 compared to 1717532474051 with source fused
06-04 16:21:14.051  6912  7032 D LocBroadcastReceiver: Duplicate location received, not sending to HA
06-04 16:22:14.028  6912  6912 D LocBroadcastReceiver: Received location update.
06-04 16:22:14.056  6912  6912 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
06-04 16:22:14.059  6912  7032 D LocBroadcastReceiver: Last Location:
06-04 16:22:14.059  6912  7032 D LocBroadcastReceiver: Coords:([redacted])
06-04 16:22:14.059  6912  7032 D LocBroadcastReceiver: Accuracy: 100.0
06-04 16:22:14.059  6912  7032 D LocBroadcastReceiver: Bearing: 0.0
06-04 16:22:14.062  6912  7032 D LocBroadcastReceiver: Begin evaluating if location update should be skipped
06-04 16:22:14.062  6912  7032 D LocBroadcastReceiver: Received location that is 94 milliseconds old, 1717532533968 compared to 1717532534062 with source fused
06-04 16:22:14.062  6912  7032 D LocBroadcastReceiver: Duplicate location received, not sending to HA
06-04 16:22:18.083  6912  6912 D IntegrationRepository: setAppActive(): false
06-04 16:22:18.107  6912  6912 D IntegrationRepository: setAppActive(): false
06-04 16:22:18.108  6912  6912 D VRI[SettingsActivity]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:18.114  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=androidx.activity.OnBackPressedDispatcher$Api34Impl$createOnBackAnimationCallback$1@d88041f
06-04 16:22:18.206  6912  6912 D VRI[SettingsActivity]: Not drawing due to not visible
06-04 16:22:20.867  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@2b30632
06-04 16:22:20.904  6912  6912 D VRI[WebViewActivity]: applyTransactionOnDraw applyImmediately
06-04 16:22:20.927  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@d2e5541
06-04 16:22:26.296  6912  5849 D WearOnboardingListener: onMessageReceived: MessageEventParcelable[17558,/request_home_assistant_instance, size=0]
06-04 16:22:26.300  6912  5849 D WearOnboardingListener: sendHomeAssistantInstance: 46ffdd4b
06-04 16:22:26.306  6912  5849 D ServerConnectionInfo: Using external URL
06-04 16:22:26.361  6912  6912 D WearOnboardingListener: sendHomeAssistantInstance: success
06-04 16:22:27.059  6912  6912 D SettingsWearAct: Waiting on Results for both connected nodes and nodes with app
06-04 16:22:27.111  6912  6912 D SettingsWearAct: Waiting on Results for both connected nodes and nodes with app
06-04 16:22:27.112  6912  6912 D SettingsWearAct: Capable Nodes: [Node{Google Pixel Watch 2, id=46ffdd4b, hops=1, isNearby=true}, Node{TicWatch E3, id=671b086b, hops=1, isNearby=true}]
06-04 16:22:27.112  6912  6912 D SettingsWearAct: Installed on all devices
06-04 16:22:27.322  6912  6912 D SettingsWearViewModel: Requesting config from node 46ffdd4b
06-04 16:22:27.323  6912  6912 D SettingsWearViewModel: Requesting config from node 671b086b
06-04 16:22:27.331  6912  6912 D SettingsWearViewModel: Request for config sent successfully
06-04 16:22:27.332  6912  6912 D SettingsWearViewModel: Request for config sent successfully
06-04 16:22:27.440  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:27.594  6912  6912 D VRI[SettingsWearActivity]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:27.596  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@7e5dee8
06-04 16:22:27.982  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:27.991  6912  6912 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: false, usesWifi is: true
06-04 16:22:27.991  6912  6912 D ServerConnectionInfo: Using external URL
06-04 16:22:27.992  6912  6912 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: false, usesWifi is: true
06-04 16:22:27.992  6912  6912 D ServerConnectionInfo: Using external URL
06-04 16:22:28.104  6912  6912 D VRI[SettingsWearMainView]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:28.120  6912  6912 D VRI[SettingsWearMainView]: Not drawing due to not visible
06-04 16:22:28.244  6912  6912 D ServerConnectionInfo: localUrl is: false, usesInternalSsid is: false, usesWifi is: true
06-04 16:22:28.244  6912  6912 D ServerConnectionInfo: Using external URL
06-04 16:22:28.246  6912  6912 D ServerConnectionInfo: localUrl is: fal:g/se, usesInternalSsid is: false, usesWifi is: true
06-04 16:22:28.246  6912  6912 D ServerConnectionInfo: Using external URL
06-04 16:22:28.290  6912  5875 D WebSocketRepository: Websocket: onOpen
06-04 16:22:28.290  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:28.303  6912  5875 D WebSocketRepository: Message number null received
06-04 16:22:28.303  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:28.303  6912  5875 D WebSocketRepository: Message number null received
06-04 16:22:28.303  6912  7202 D WebSocketRepository: Auth Requested
06-04 16:22:28.306  6912  6912 D WebSocketRepository: Sending message 1: {type=supported_features, id=1, features={coalesce_messages=1}}
06-04 16:22:28.307  6912  6912 D WebSocketRepository: Sending message 2: {type=get_states, id=2}
06-04 16:22:28.307  6912  6912 D WebSocketRepository: Message number 2 sent
06-04 16:22:28.335  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:28.335  6912  5875 D WebSocketRepository: Message number 1 received
06-04 16:22:28.410  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:28.423  6912  5875 D WebSocketRepository: Message number 2 received
06-04 16:22:29.249  6912  5877 D WearOnboardingListener: onMessageReceived: MessageEventParcelable[17560,/request_home_assistant_instance, size=0]
06-04 16:22:29.249  6912  5877 D WearOnboardingListener: sendHomeAssistantInstance: 46ffdd4b
06-04 16:22:29.250  6912  5877 D ServerConnectionInfo: Using external URL
06-04 16:22:29.256  6912  6912 D WearOnboardingListener: sendHomeAssistantInstance: success
06-04 16:22:30.201  6912  6912 I ImeTracker: io.homeassistant.companion.android:63328c47: onRequestShow at ORIGIN_CLIENT_SHOW_SOFT_INPUT reason SHOW_SOFT_INPUT
06-04 16:22:30.201  6912  6912 D InputMethodManager: showSoftInput() view=android.webkit.WebView{4068f2 VFEDHVC.. .F...... 0,0-1080,2200 aid=1073741824} flags=0 reason=SHOW_SOFT_INPUT
06-04 16:22:30.210  6912  6912 I AssistStructure: Flattened final assist data: 3628 bytes, containing 1 windows, 9 views
06-04 16:22:30.472  6912  6912 D InsetsController: show(ime(), fromIme=true)
06-04 16:22:30.684  6912  6912 I ImeTracker: io.homeassistant.companion.android:63328c47: onShown
06-04 16:22:31.889  6912  6912 I ImeTracker: io.homeassistant.companion.android:347b727: onRequestShow at ORIGIN_CLIENT_SHOW_SOFT_INPUT reason SHOW_SOFT_INPUT
06-04 16:22:31.889  6912  6912 D InputMethodManager: showSoftInput() view=android.webkit.WebView{4068f2 VFEDHVC.. .F...... 0,0-1080,1311 aid=1073741824} flags=0 reason=SHOW_SOFT_INPUT
06-04 16:22:31.893  6912  6912 D InsetsController: show(ime(), fromIme=true)
06-04 16:22:31.893  6912  6912 I ImeTracker: io.homeassistant.companion.android:347b727: onCancelled at PHASE_CLIENT_APPLY_ANIMATION
06-04 16:22:42.959  6912  6912 I ImeTracker: io.homeassistant.companion.android:d619bee1: onRequestHide at ORIGIN_CLIENT_HIDE_SOFT_INPUT reason HIDE_SOFT_INPUT
06-04 16:22:42.965  6912 23932 W RemoteInputConnectionImpl: requestCursorUpdates on inactive InputConnection
06-04 16:22:42.985  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=ImeCallback=ImeOnBackInvokedCallback@146441450 Callback=android.window.IOnBackInvokedCallback$Stub$Proxy@a676dec
06-04 16:22:42.999  6912  6912 W Choreographer: Frame time is 0.0725 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
06-04 16:22:42.999  6912  5890 W Choreographer: Frame time is 0.072448 ms in the future!  Check that graphics HAL is generating vsync timestamps using the correct timebase.
06-04 16:22:43.190  6912  6912 I ImeTracker: io.homeassistant.companion.android:6080398f: onRequestHide at ORIGIN_CLIENT_HIDE_SOFT_INPUT reason HIDE_SOFT_INPUT_BY_INSETS_API
06-04 16:22:43.191  6912  6912 I ImeTracker: io.homeassistant.companion.android:d619bee1: onHidden
06-04 16:22:43.766  6912  6912 I ImeTracker: io.homeassistant.companion.android:8dd35143: onRequestHide at ORIGIN_CLIENT_HIDE_SOFT_INPUT reason HIDE_SOFT_INPUT_CLOSE_CURRENT_SESSION
06-04 16:22:45.434  6912  6912 D VRI[SettingsWearMainView]: Start draw after previous draw not visible
06-04 16:22:45.461  6912  6912 D SettingsWearViewModel: Successfully sent auth to wear
06-04 16:22:45.468  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:45.989  6912  6912 D VRI[OnboardingActivity]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:45.996  6912 23310 D HWUI    : endAllActiveAnimators on 0x6f5cdfc8a0 (UnprojectedRipple) with handle 0x6f7cd54a60
06-04 16:22:46.007  6912  6912 D VRI[OnboardingActivity]: Not drawing due to not visible
06-04 16:22:46.013  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=androidx.activity.OnBackPressedDispatcher$Api34Impl$createOnBackAnimationCallback$1@ba721d1
06-04 16:22:46.019  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@f7bd8f8
06-04 16:22:46.536  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: Watch was unable to register: javax.net.ssl.SSLHandshakeException: connection closed
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at com.android.org.conscrypt.SSLUtils.toSSLHandshakeException(SSLUtils.java:363)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at com.android.org.conscrypt.ConscryptEngineSocket.doHandshake(ConscryptEngineSocket.java:243)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at com.android.org.conscrypt.ConscryptEngineSocket.startHandshake(ConscryptEngineSocket.java:220)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealConnection.connectTls(RealConnection.kt:379)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealConnection.establishProtocol(RealConnection.kt:337)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:209)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:226)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:106)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:74)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealCall.initExchange$okhttp(RealCall.kt:255)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:32)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:95)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:83)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:76)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:109)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(RealCall.kt:201)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(RealCall.kt:517)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1137)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:637)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	at java.lang.Thread.run(Thread.java:1012)
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: Caused by: java.io.EOFException: connection closed
06-04 16:22:46.538  6912  6912 E SettingsWearViewModel: 	... 22 more
06-04 16:22:46.551  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:46.737  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:46.757  6912  6912 D SettingsWearViewModel: onDataChanged 1
06-04 16:22:49.744  6912  6912 D VRI[SettingsWearMainView]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:49.757  6912 23310 D HWUI    : endAllActiveAnimators on 0x6f5cd9f320 (UnprojectedRipple) with handle 0x6f7cd72fd0
06-04 16:22:49.770  6912  6912 D VRI[SettingsWearMainView]: Not drawing due to not visible
06-04 16:22:49.812  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@3ea5345
06-04 16:22:49.866  6912  5875 D WebSocketRepository: Websocket: onClosing code: 1000, reason:
06-04 16:22:49.881  6912  5875 D WebSocketRepository: Websocket: onClosed
06-04 16:22:50.999  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:50.999  6912  6912 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.001  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.002  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.002  6912  6912 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.003  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.013  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.050  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.063  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.135  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.135  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 0, currentMillis: 1717532571135)
06-04 16:22:51.141  6912  6912 D WebviewActivity: Current webview package com.android.webview and version 125.0.6422.113
06-04 16:22:51.142  6912  6912 D WebviewActivity: Matter/Thread step changed to NOT_STARTED
06-04 16:22:51.144  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.146  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.146  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 0, currentMillis: 1717532571146)
06-04 16:22:51.245  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.267  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.267  6912  7205 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.272  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.272  6912  7205 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.287  6912  5875 D WebSocketRepository: Websocket: onOpen
06-04 16:22:51.288  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:51.293  6912  5875 D WebSocketRepository: Message number null received
06-04 16:22:51.294  6912  7205 D WebSocketRepository: Auth Requested
06-04 16:22:51.294  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:51.294  6912  5875 D WebSocketRepository: Message number null received
06-04 16:22:51.295  6912  7205 D WebSocketRepository: Sending message 1: {type=supported_features, id=1, features={coalesce_messages=1}}
06-04 16:22:51.296  6912  7205 D WebSocketRepository: Sending message 2: {type=auth/current_user, id=2}
06-04 16:22:51.296  6912  7205 D WebSocketRepository: Message number 2 sent
06-04 16:22:51.303  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:51.304  6912  5875 D WebSocketRepository: Message number 1 received
06-04 16:22:51.309  6912  5875 D WebSocketRepository: Websocket: onMessage (text)
06-04 16:22:51.310  6912  5875 D WebSocketRepository: Message number 2 received
06-04 16:22:51.350  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.351  6912  7077 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=84539ed1-dc54-42a2-9a55-a9e8ce0c5725, tags={ io.homeassistant.companion.android.websocket.WebsocketManager } ]
06-04 16:22:51.362  6912  7035 I WM-Processor: Moving WorkSpec (f47442d3-863d-4df5-81f9-1435ae624a2b) to the foreground
06-04 16:22:51.365  6912  6912 I WM-SystemFgDispatcher: Started foreground service Intent { act=ACTION_START_FOREGROUND cmp=io.homeassistant.companion.android/androidx.work.impl.foreground.SystemForegroundService (has extras) }
06-04 16:22:51.367  6912 23926 D SensorWorker: Updating all Sensors in foreground.
06-04 16:22:51.368  6912 23926 D LastUpdate: Last update is SensorWorker
06-04 16:22:51.373  6912 23926 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.374  6912 23926 D ActivitySM: Unregistering for activity updates.
06-04 16:22:51.374  6912 23926 D ActivitySM: Registering for activity updates.
06-04 16:22:51.500  6912  6912 D VRI[LaunchActivity]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:51.513  6912  6912 D VRI[LaunchActivity]: Not drawing due to not visible
06-04 16:22:51.514  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=android.app.Activity$$ExternalSyntheticLambda0@a99fbab
06-04 16:22:51.530  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.530  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 0, currentMillis: 1717532571530)
06-04 16:22:51.541  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.541  6912  6912 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.543  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.543  6912  6912 D ServerConnectionInfo: Using internal URL
06-04 16:22:51.603  6912 23926 D LightSensor: Light sensor listener registered
06-04 16:22:51.610  6912  6912 D LightSensor: Light sensor listener unregistered
06-04 16:22:51.628  6912  6912 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
06-04 16:22:51.648  6912 23926 D NextAlarm: No alarm is scheduled, sending unavailable
06-04 16:22:51.689  6912 23926 D ProximitySensor: Proximity sensor listener registered
06-04 16:22:51.697  6912 23926 D StepsSensor: Steps sensor listener registered
06-04 16:22:51.704  6912  6912 D StepsSensor: Steps sensor listener unregistered
06-04 16:22:51.712  6912 23926 D StorageSensor: PATHS SD 1
06-04 16:22:51.717  6912 23926 D StorageSensor: PATH /storage/emulated/0/Android/data/io.homeassistant.companion.android/files is mounted true and removable is false
06-04 16:22:51.734  6912  6912 D ProximitySensor: Proximity sensor listener unregistered
06-04 16:22:51.742  6912 23926 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.744  6912 23926 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.809  6912 23926 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:51.954  6912  7032 I SensorReceiver: Sensor updates and sync completed
06-04 16:22:51.955  6912  7035 I WM-WorkerWrapper: Worker result SUCCESS for Work [ id=f47442d3-863d-4df5-81f9-1435ae624a2b, tags={ io.homeassistant.companion.android.sensors.SensorWorker } ]
06-04 16:22:51.961  6912  6912 I WM-SystemFgDispatcher: Stopping foreground service
06-04 16:22:52.052  6912 24488 D WebviewActivity: External bus {"type":"config/get","id":1}
06-04 16:22:52.054  6912  6912 D WebviewActivity: externalBus({"id":1,"type":"result","success":true,"result":{"hasSettingsScreen":true,"canWriteTag":true,"hasExoPlayer":true,"canCommissionMatter":true,"canImportThreadCredentials":true,"hasAssist":true}});
06-04 16:22:52.646  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:52.646  6912  6912 D ServerConnectionInfo: Using internal URL
06-04 16:22:52.655  6912  6912 D WebviewActivity: Callback null
06-04 16:22:52.655  6912  6912 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
06-04 16:22:52.655  6912  6912 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
06-04 16:22:52.655  6912  7032 D WebViewPresenterImpl: Try getting color from webview color "#101e24".
06-04 16:22:52.655  6912  7032 I WebViewPresenterImpl: Found color -15720924.
06-04 16:22:52.656  6912  7202 D WebViewPresenterImpl: Try getting color from webview color "#111111".
06-04 16:22:52.656  6912  7202 I WebViewPresenterImpl: Found color -15658735.
06-04 16:22:52.718  6912 24488 D WebviewActivity: External bus {"type":"connection-status","payload":{"event":"connected"},"id":2}
06-04 16:22:52.722  6912  6912 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:52.828  6912 24488 D WebviewActivity: External bus {"type":"theme-update","id":3}
06-04 16:22:52.840  6912  6912 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
06-04 16:22:52.840  6912  6912 D WebviewActivity: Color from webview is "#101e24-SPACER-#111111"
06-04 16:22:52.840  6912  7202 D WebViewPresenterImpl: Try getting color from webview color "#101e24".
06-04 16:22:52.841  6912  7202 I WebViewPresenterImpl: Found color -15720924.
06-04 16:22:52.841  6912  7032 D WebViewPresenterImpl: Try getting color from webview color "#111111".
06-04 16:22:52.841  6912  7032 I WebViewPresenterImpl: Found color -15658735.
06-04 16:22:53.697  6912 23798 I cr_MediaCodecBridge: create MediaCodec video decoder, mime video/avc, decoder name OMX.qcom.video.decoder.avc
06-04 16:22:53.752  6912  6094 I OMXClient: IOmx service obtained
06-04 16:22:53.824  6912  6092 I MediaCodec: MediaCodec will operate in async mode
06-04 16:22:53.826  6912  6092 D SurfaceUtils: connecting to surface 0x703ce73c90, reason connectToSurface
06-04 16:22:53.827  6912  6092 I MediaCodec: [OMX.qcom.video.decoder.avc] setting surface generation to 7077893
06-04 16:22:53.827  6912  6092 D SurfaceUtils: disconnecting from surface 0x703ce73c90, reason connectToSurface(reconnect)
06-04 16:22:53.827  6912  6092 D SurfaceUtils: connecting to surface 0x703ce73c80, reason connectToSurface(reconnect-with-listener)
06-04 16:22:53.832  6912 23798 D MediaCodec: keep callback message for reclaim
06-04 16:22:53.857  6912  6094 D SurfaceUtils: set up nativeWindow 0x703ce73c90 for 1280x720, color 0x7fa30c06, rotation 0, usage 0x20402900
06-04 16:22:57.298  6912 23798 W cr_MediaCodecBridge: Releasing: OMX.qcom.video.decoder.avc
06-04 16:22:57.298  6912  6092 D SurfaceUtils: connecting to surface 0x703ce75ea0, reason connectToSurface
06-04 16:22:57.298  6912  6092 I MediaCodec: [OMX.qcom.video.decoder.avc] setting surface generation to 7077894
06-04 16:22:57.298  6912  6092 D SurfaceUtils: disconnecting from surface 0x703ce75ea0, reason connectToSurface(reconnect)
06-04 16:22:57.298  6912  6092 D SurfaceUtils: connecting to surface 0x703ce75e90, reason connectToSurface(reconnect-with-listener)
06-04 16:22:57.299  6912  6094 D SurfaceUtils: set up nativeWindow 0x703ce75ea0 for 1280x720, color 0x7fa30c06, rotation 0, usage 0x20402900
06-04 16:22:57.299  6912  6092 D SurfaceUtils: disconnecting from surface 0x703ce73c90, reason disconnectFromSurface
06-04 16:22:57.300  6912  6092 E Surface : freeAllBuffers: 12 buffers were freed while being dequeued!
06-04 16:22:57.300  6912 23798 W cr_MediaCodecBridge: Codec released
06-04 16:22:57.325  6912  6092 D SurfaceUtils: disconnecting from surface 0x703ce75ea0, reason disconnectFromSurface
06-04 16:22:59.069  6912 24488 D WebviewActivity: External bus {"type":"config_screen/show","id":4}
06-04 16:22:59.094  6912  6912 D IntegrationRepository: setAppActive(): false
06-04 16:22:59.098  6912  6912 D IntegrationRepository: setAppActive(): false
06-04 16:22:59.108  6912  6912 D SensorReceiver: Received intent: io.homeassistant.companion.android.UPDATE_SENSORS
06-04 16:22:59.112  6912  6912 D LastUpdate: Last update is io.homeassistant.companion.android.UPDATE_SENSORS
06-04 16:22:59.120  6912  7202 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:59.122  6912  7202 D ActivitySM: Unregistering for activity updates.
06-04 16:22:59.122  6912  7202 D ActivitySM: Registering for activity updates.
06-04 16:22:59.201  6912  7202 D LightSensor: Light sensor listener registered
06-04 16:22:59.237  6912  7202 D NextAlarm: No alarm is scheduled, sending unavailable
06-04 16:22:59.256  6912  7202 D ProximitySensor: Proximity sensor listener registered
06-04 16:22:59.259  6912  7202 D StepsSensor: Steps sensor listener registered
06-04 16:22:59.266  6912  7202 D StorageSensor: PATHS SD 1
06-04 16:22:59.269  6912  7202 D StorageSensor: PATH /storage/emulated/0/Android/data/io.homeassistant.companion.android/files is mounted true and removable is false
06-04 16:22:59.276  6912  7202 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:59.277  6912  7202 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:59.289  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 0, currentMillis: 1717532579289)
06-04 16:22:59.296  6912  6912 D LightSensor: Light sensor listener unregistered
06-04 16:22:59.297  6912  6912 D StepsSensor: Steps sensor listener unregistered
06-04 16:22:59.298  6912  6912 D ProximitySensor: Proximity sensor listener unregistered
06-04 16:22:59.356  6912  7202 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:59.370  6912 23926 I SensorReceiver: Sensor updates and sync completed
06-04 16:22:59.379  6912  7205 D ServerConnectionInfo: localUrl is: true, usesInternalSsid is: true, usesWifi is: true
06-04 16:22:59.398  6912  6912 D ForegrndServiceLauncher: Check if service HighAccuracyLocationService is running. Service running = false
06-04 16:22:59.499  6912  6912 D IntegrationRepository: isAppLocked(): false. (LockEnabled: false, appActive: false, expireMillis: 0, currentMillis: 1717532579499)
06-04 16:22:59.499  6912  6912 D IntegrationRepository: setAppActive(): true
06-04 16:22:59.888  6912  6912 D VRI[WebViewActivity]: visibilityChanged oldVisibility=true newVisibility=false
06-04 16:22:59.912  6912  6912 D VRI[WebViewActivity]: Not drawing due to not visible
06-04 16:22:59.937  6912  6912 W WindowOnBackDispatcher: sendCancelIfRunning: isInProgress=falsecallback=androidx.activity.OnBackPressedDispatcher$Api34Impl$createOnBackAnimationCallback$1@a47aaeb
06-04 16:23:03.287  6912 23926 D LogcatReader: Read logcat for pid 6912

```

**Screenshot or video of problem:**

**Additional information:**
My HA is configured to use HTTPS (SSL/TLS) with a Let's Encrypt cert.

**Comment by dshokouhi** [comment]
Unfortunately SSL errors are not ones the app can solve, can you try maybe regenerating your certificate?

**Comment by brianjmurrell** [comment]
The certificate is fine.  The browser and Android app have not problem with it.

So this is not a _certificate_ problem as in an invalid certificate in any way.

**Comment by dshokouhi** [comment]
There may not be an issue with the certificate but there is nothing the app can do if your device (watch) does not want to accept it. Maybe you can try restarting things on the network to see if it can work but from the app sde we cant do anything but make these suggestions.

**Comment by brianjmurrell** [comment]
Note that at no point in time during the process of trying to connect my watch to my HA (through logging in on the phone) does the watch even try to connect to my HA so how is the certificate on my HA even relevant?

**Comment by dshokouhi** [comment]
We can only go by the errors in the logs so something on your network may be trying to close the connection? I am not really sure but calls made on the watch use the same code base as the phone. They often use the same API and communication method. Based on the symptoms I would look at things on the network like proxy etc... Maybe something just needs to be restarted?

**Comment by brianjmurrell** [comment]
Ah ha!  I was finally able to connect it.  I don't really know how or why.  The only thing I did do was disable BlueTooth on it.  But I think it still failed a time or two after that.

Bottom line though is that the error being reported here **IS NOT** due to the certificate as you said otherwise it would have never worked and it did eventually.

So there is definitely a problem with the app on the watch and/or phone here that causes this problem and this issue should remain open until that is resolved.

**Comment by dshokouhi** [comment]
> The only thing I did do was disable BlueTooth on it

that would be this issue #1866 




> So there is definitely a problem with the app on the watch and/or phone here that causes this problem and this issue should remain open until that is resolved.

its a network error that the app received here as evidenced by you turning off bluetooth and using Wifi. Wear OS does some proxy magic over bluetooth. This is not an issue the app can solve. Not to mention not every user hits this issue.

as of now this seems like a duplicate of the mentioned issue even though the logs may not match the other users in that thread

**Comment by dshokouhi** [comment]
closing as duplicate of #1866 since we have a lot of research in that issue. Multiple causes stemming to the same symptoms of turning off bluetooth
